data/README.md

```diff
@@ -1,5 +1,5 @@
-Unzip the artifacts here. You'll end up with:
+Download artifacts here.
```

Comment:
Can you add the other files that will be downloaded from huggingface - the training data (split).

Reply:
Ok I will add an explanation that artifacts will be downloaded automatically into `~/.cache/huggingface/datasets`

and the train/validation/test split is happening now in the code